extensions/amp-story/amp-story.md

```diff
@@ -720,6 +728,37 @@ An `amp-story-grid-layer` can contain any of the following elements:
   </tr>
 </table>
 
+#### Clickable elements and embeds in amp-story-grid-layer
```

Comment:
This might be hard to understand for someone with no context on our project. How about something like "Links and AMP components in pages"?
I think this documentation is about how to use links and other AMP components within a story, we should try to make it as clear as possible. 
What do you think of rephrasing the next few sentences to better reflect that? I don't think we have to justify our product decisions in the documentation ("this is because we want..."), but rather communicate on what use cases the feature allow.

Reply:
I agree that "clickable" might not be the best wording, since the target users will be on mobile and "tapping" instead of clicking. My vote is for "User Interactive Elements in amp-story-grid-layer"

Reply:
One thing to note is that an upcoming iteration will introduce an option to make these elements either interactive or not interactive. Do you think it'd make sense to simply talk about links and AMP components (which could be interactive, or not)?

Reply:
I went ahead and used "User Interactive Elements" since this section has to do with the tooltip customization - which only has to do with interactive elements.

I can add a new section later for the embedded components where we explain the difference between interactive and non-interactive embedded components.

Reply:
What's our naming strategy here?

Right now we have:
1. User interactive elements in amp-story-grid-layer
2. Customizing tooltip for tappable elements
2.a `data-tooltip-icon`
2.b `data-tooltip-text`

So what will 3 be? I don't believe "User non interactive elements" make sense.

Our goals should be: making what you can use, and how to use it very clear. Right now if I read the documentation as a new publisher, how am I supposed to know what's a "user interactive element"?

I was suggesting something like:

1. Links and AMP components in amp-story-grid-layer
We support inline links `<a>` and AMP components (<amp-instagram>, <amp-twitter>). They can be interactive or not (default). If interactive, tooltip is displayed on user interaction.
2. Customizing tooltip for tappable elements
2.a `data-tooltip-icon`
2.b `data-tooltip-text`

And you don't need a 3.

feel free to push back if you think I'm being too crazy here

Reply:
SGTM, but I decided to just keep the part about links since the interactive components aren't yet live. I'll update it when the validator stuff hits prod.